pubspec.lock

```diff
@@ -7,42 +7,42 @@ packages:
       name: async
```

Comment:
never add pubspec.lock to the commit unless you have changed the pubspec.yaml. Revert this to the older version and then I'll merge it.

Reply:
Should I replace .lock file with original or try something else for this ?

Reply:
commit 873354c  was unintentional will be resolved

Reply:
> Should I replace .lock file with original or try something else for this ?

yes. replace it with the original one.